changedetectionio/forms.py

```diff
@@ -338,6 +338,8 @@ class watchForm(commonSettingsForm):
     ignore_status_codes = BooleanField('Ignore status codes (process non-2xx status codes as normal)', default=False)
     trigger_text = StringListField('Trigger/wait for text', [validators.Optional(), ValidateListRegex()])
 
+    sync_filters_across_tags = BooleanField('Copy filter/trigger settings to all watches with matching watch groups')
```

Comment:
what about a small icon in the [tab] in the page list to indicate thats being 'synced' ?

Reply:
maybe better to say "with the _same_ watch groups"